Wrappers/Python/cil/framework/framework.py

```diff
@@ -1116,7 +1119,10 @@ def __eq__(self, other):
         return False
 
     def calculate_magnification(self):
```

Comment:
Hi Jakob, this method is different for parallel beam because we weren't previously calculating the source to object distance or the object to detector distance and were instead just setting the magnification = 1. I changed it to also calculate the object to detector distance so we can use it in the Paganin calculation. The cone beam one stays the same. The phase retrieval is different for cone and parallel because the equation uses the magnification and propagation distance. I'm not sure about fan beam, but as long as the geometry contains magnification and dist_center_detector it should work.